frontend/src/cards/MapCard.tsx

```diff
@@ -138,86 +174,75 @@ function MapCardWithKey(props: MapCardProps) {
+                        options={filterOptions}
+                        onOptionUpdate={(option) =>
+                          setActiveBreakdownFilter(option)
+                        }
+                      />
                     </Grid>
                     <Grid item>
-                      {/* TODO- Clean up UI */}
-                      <List component="nav">
-                        <ListItem button onClick={popover.open}>
-                          <ListItemText primary={breakdownFilter} />
-                          <ArrowDropDownIcon />
-                        </ListItem>
-                      </List>
-                      {/* TODO - Align this with the mocks */}
-                      <Menu
-                        anchorEl={popover.anchor}
-                        keepMounted
-                        open={popover.isOpen}
-                        onClose={popover.close}
+                      <Button
+                        onClick={() => setSmallMultiplesDialogOpen(true)}
+                        color="primary"
                       >
-                        {["age", "all"].includes(props.currentBreakdown) && (
-                          <MenuItem disabled={true}>Age [unavailable]</MenuItem>
-                        )}
-                        {["sex", "all"].includes(props.currentBreakdown) && (
-                          <MenuItem disabled={true}>Sex [unavailable]</MenuItem>
-                        )}
-                        {["race_and_ethnicity", "all"].includes(
-                          props.currentBreakdown
-                        ) && (
-                          <>
-                            <MenuItem disabled={true}>Races</MenuItem>
-                            {breakdownValues.map((option) => (
-                              <MenuItem
-                                key={option}
-                                onClick={(e) => {
-                                  popover.close();
-                                  setBreakdownFilter(option);
-                                }}
-                              >
-                                {option}
-                              </MenuItem>
-                            ))}
-                          </>
-                        )}
-                      </Menu>
+                        Show full breakdown by{" "}
+                        {BREAKDOWN_VAR_DISPLAY_NAMES[activeBreakdown]}
+                      </Button>
                     </Grid>
                   </Grid>
                 </CardContent>
               </>
             )}
             <Divider />
-            <CardContent>
-              {queryResponse.dataIsMissing() && (
+            {queryResponse.dataIsMissing() && (
+              <CardContent>
                 <Alert severity="error">No data available</Alert>
+              </CardContent>
+            )}
+            {!queryResponse.dataIsMissing() &&
+              dataForActiveBreakdownFilter.length === 0 && (
```

Comment:
What's the use case for this condition? Sorry, I'm having trouble following the different data-missing scenarios

Reply:
If there is some data, but not for the currently selected filter. For instance if you select Unknown there is no field data.